secret_santa.py

```diff
@@ -105,30 +106,30 @@ def main(argv=None):
         participants = config['PARTICIPANTS']
         if len(participants) < 2:
             raise Exception('Not enough participants specified.')
-        
+
         givers = []
         for person in participants:
-            name, email, amazon = re.match(r'([^<]*)<([^>]*)>.*(http.*)', person).groups()
+            name, email, amazon, country = re.match(r'([^<]*)<([^>]*)>.*(http.*) ([A-z]{2})', person).groups()
             name = name.strip()
```

Comment:
Hmm okay, the reason why I didn't use country name was because it felt tedious to write "United States of America", all over the yaml file.